guide/daily-spending-wallet/security.md

```diff
@@ -226,29 +201,7 @@ There may be situations in which users do not want to show their real balances a
 
 To further prevent unwanted spending, wallet limits can be established. These can limit the size of single transactions, or impose daily limits. Limits should be changeable at any time, but they should only apply after a certain amount of time has passed. Alternatively, a dedicated password could be required to change limits instantly. So even if a third-party manages to access the wallet, the pace at which they could steal the funds would be drastically slowed down.
 
```

Comment:
The grey background used in the Privacy & Security settings image in Wallet limits section is inconsistent with other frames in the UI kit / has bad contrast with the background of the gallery, could make it white or a lighter grey?

Reply:
This also applies to several images in below this one in watchtowers and deleting the recovery phrase sections